src/main/java/org/fundacionjala/coding/abel/movies/Movie.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.coding.abel.movies;
+
+/**
+ * Created by Unknown on 5/26/2017.
+ */
+public class Movie {
+
+    public static final int CHILDREN = 2;
+    public static final int REGULAR = 0;
+    public static final int NEW_RELEASE = 1;
+    private String title;
+    private int priceCode;
+
+    /**
+     * This is the default constructor.
+     * @param title String.
+     * @param priceCode int.
+     */
+    public Movie(String title, int priceCode) {
+        this.title = title;
+        this.priceCode = priceCode;
```

Comment:
You need to take advantage of inheritance.
so you need to remove the priceCode dependency in this case.